dspace-api/src/test/data/dspaceFolder/config/local.cfg

```diff
@@ -120,3 +120,6 @@ rest.properties.exposed = configuration.not.existing
 configuration.not.exposed = secret_value
 configuration.exposed.single.value = public_value
 configuration.exposed.array.value = public_value_1, public_value_2
+
+orcid.clientid = 
+orcid.clientsecret = 
```

Comment:
These should not be needed.  Again, these can be set directly as environment variables, so you don't need to insert them here so that they can be replaced later.